print(1)

try:
	a= 1/0
except:
	print('i failed')




print(0)